addons/mail/security/mail_security.xml

```diff
@@ -266,7 +266,7 @@
         <record id="res_users_settings_volumes_rule_admin" model="ir.rule">
             <field name="name">Administrators can access all User Settings volumes.</field>
             <field name="model_id" ref="model_res_users_settings_volumes"/>
-            <field name="groups" eval="[Command.link(ref('base.group_system'))]"/>
+            <field name="groups" eval="[Command.link(ref('base.group_erp_manager'))]"/>
```

Comment:
Hello, I'm not sure why you need to make the change, but if you do, you'll need an upgrade script to force update the rule.

Reply:
AFAIR the group group_erp_manager can create a user but not a res.users.settings.

This was an issue when adding the settings in user values but i could change that by a sudo, will try that now to avoid loosing to much time with a upgrade script.